libs/hdf-converters/test/attestations/attestations.spec.ts

```diff
@@ -110,4 +110,28 @@ describe('attestations', () => {
       advanceDate(moment.utc(1662758942000), 'fortnightly').toISOString(true)
     ).toEqual('2022-09-23T21:29:02.000+00:00');
   });
+
+  it('Should parse input for custom number of days correctly', () => {
```

Comment:
Can you add clauses in the attestation array that makes it clear the user input will be understood as expected?